business_requirement_deliverable_cost/models/business.py

```diff
@@ -57,14 +57,26 @@ def _get_partner(self):
             return False
 
     @api.multi
+    def _get_project(self):
+        self.ensure_one()
+        br_id = br_deliverable = False
```

Comment:
Since those variables are only used inside the `if`s, I think there's no need to init them here